LifeCodeLabs/Moduls/moduls_filter_fastq.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+
+def read_fastq(input_fastq):
+    with open(input_fastq, 'r') as file:
+        f = file.readlines()
+
+    fast = {}
+    for index in range(0,len(f),4):
```

Comment:
Хорошее решение!

Reply:
```suggestion
    for index in range(0, len(f) ,4):
```
Только бы пробелы добавить